src/main/java/com/hotels/beeju/core/BeejuCore.java

```diff
@@ -101,6 +101,10 @@ void setHiveVar(HiveConf.ConfVars variable, String value) {
     conf.setVar(variable, value);
   }
 
+  void setHiveConf(String variable, String value) {
```

Comment:
OK, so this allows one to set arbitrary String values in the HiveConf as opposed to only the ones that Hive itself exposes via `HiveConf.ConfVars`. It looks like a reasonable addition to me and I can see how this could be useful. Could you please add a simple unit test for this along the lines of https://github.com/HotelsDotCom/beeju/blob/master/src/test/java/com/hotels/beeju/core/BeejuCoreTest.java#L75?

Reply:
Good call, I made the changes.